### DISCUSSION TITLE
PDF file of schematics?

### DISCUSSION THREAD

**Discussion by Zitt** [bitfieldlabs/afterglow#16] [Q&A]
Can someone print the schematics of the KiCad to PDF and upload it to the DOCs section?
Unless I missed it.

**Comment by bitfieldlabs** [comment]
A plot of the schematic for v1.5 is now here:
https://github.com/bitfieldlabs/afterglow/blob/master/afterglow_pcb_v15/afterglow_nano_smd.pdf
